deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -10133,6 +10133,41 @@ spec:
                   items:
                     type: string
                   type: array
+                auth:
+                  description: The authentication configuration
+                  properties:
+                    keystone:
+                      description: The spec for Keystone
+                      nullable: true
+                      properties:
+                        acceptedRoles:
+                          description: The roles requires to serve requests.
+                          items:
+                            type: string
+                          type: array
+                        implicitTenants:
+                          description: Create new users in their own tenants of the same name. Possible values are true, false, swift and s3. The latter have the effect of splitting the identity space such that only the indicated protocol will use implicit tenants.
+                          type: string
+                        revocationInterval:
```

Comment:
I don't see that this value is used anywhere to configure rgw. Also in my CEPH env I cannot set this option:
```
Error EINVAL: unrecognized config option 'rgw_keystone_revocation_interval'
```

Reply:
I assume there isn't a replacement config either? I don't see anything similar in current docs.

Reply:
It was removed with this commit https://github.com/ceph/ceph/commit/6b4c985319224b3fe34f618b3faa07b183b055c8 
If I understand the commit message correctly there was no need for a replacement.